docs/analyzers/Using Additional Files.md

```diff
@@ -0,0 +1,231 @@
+Introduction
+============
+
+This document covers the following:
+
+* Uses of Additional Files
+* Passing Additional Files on the command line
+* Specifying an individual item as an `AdditionalFile` in an MSBuild project
+* Specifying an entire set of items as `AdditionalFile`s in an MSBuild project
+* Accessing and reading additional files through the `AnalyzerOptions` type
+* Includes sample analyzers that read additional files
+
+Uses
+====
+
+Sometimes an analyzer needs access to information that is not available through normal compiler inputs--source files, references, and options. To support these scenarios the C# and Visual Basic compilers can accept additional, non-source text files as inputs.
+
+On the Command Line
+===================
+
+On the command line, additional files can be passed using the `/additionalfile` option. For example:
+```
+csc.exe alpha.cs /additionalfile:terms.txt
+```
+
+In a Project File
+=================
+
+Passing an Individual File
+--------------------------
+
+To specify an individual project item as an additional file, set the item type to `AdditionalFile`:
+
+``` XML
+<ItemGroup>
+  <AdditionalFile Include="terms.txt" />
```

Comment:
I think this needs to be AdditionalFiles (plural) since that's what's passed to the csc task. Btw I have never tried adding an additional file like this. Can you make sure to test that this works?